test/e2e/storage/subpath.go

```diff
@@ -271,6 +271,18 @@ var _ = utils.SIGDescribe("Subpath", func() {
 				}
 				testSubpathReconstruction(f, pod, true)
 			})
+
+			It("should support readOnly specified in the volumeMount", func() {
```

Comment:
Also need test cases for when the volume source is read only.  For inline volumes and PV volumes.

Reply:
added